libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -553,9 +563,9 @@ void AP_MotorsMulticopter::output_logic()
         _spool_state = SpoolState::SHUT_DOWN;
     }
 
-    if (_spool_up_time < 0.05) {
+    if (_spool_up_time < minimum_spool_time) {
```

Comment:
Our normal convention is to use = 0 to turn off a parameter. We may have to remove the _spool_up_time.set(minimum_spool_time) and just use the minimum value if it is zero.
That way if we use _spool_down_time = 0 we can use _spool_up_time.

Reply:
This part isn't a change of behavior at the moment. I'm not opposed to changing this, but I think it's somewhat fair that this is actually changing the parameter and the user will have a chance to see see it when they download parameters. 

Spool up time was previous and had to be enabled. Are you saying you want a change in behavior on spool up time?

Reply:
Yes, I agree.